pkg/operation/botanist/logging.go

```diff
@@ -39,9 +50,19 @@ func (b *Botanist) DeploySeedLogging(ctx context.Context) error {
 		return err
 	}
 
+	hvpaValues := make(map[string]interface{})
+	hvpaEnabled := gardenletfeatures.FeatureGate.Enabled(features.HVPA)
+	if b.ShootedSeed != nil {
+		hvpaEnabled = gardenletfeatures.FeatureGate.Enabled(features.HVPAForShootedSeed)
+	}
+
+	hvpaValues["enabled"] = hvpaEnabled
+	hvpaValues["maintenanceWindow"] = b.Shoot.Info.Spec.Maintenance.TimeWindow
```

Comment:
I don't think we need the maintenance time window for the Loki scaling, it's not that critical like etcd. WDYT? cc @amshuman-kr

Reply:
I am okay to remove the check

Reply:
Yes. Not required here.

Reply:
Okay, I will remove it :)

Reply:
We can remove this as well as change scaleDown `updateMode` to `Auto` as mentioned above.

Reply:
It should be removed now